nucypher/blockchain/eth/interfaces.py

```diff
@@ -262,12 +264,7 @@ def get_blocktime(self):
 
     @property
     def is_connected(self) -> bool:
-        """
-        https://web3py.readthedocs.io/en/stable/__provider.html#examples-using-automated-detection
-        """
-        if self.client is NO_BLOCKCHAIN_CONNECTION:
-            return False
-        return self.client.is_connected
+        return self.__is_connected
```

Comment:
```suggestion
        return self._is_connected
```

Reply:
I'm not going to go through them all because I see that David has suggestion on the naming of this variable too.

Reply:
See response in other comment.